css/global.scss

```diff
@@ -0,0 +1 @@
+@include icon-black-white('notes', 'notes', 1);
```

Comment:
This is a full request for just your icon? :( 
Use the dashboard template render event please

Reply:
Baahm! But it's *my* icon! :laughing: 

To be honest: I just took this approach from the deck app, see
https://github.com/nextcloud/deck/blob/v1.1.0-beta2/lib/AppInfo/Application20.php#L87 and
https://github.com/nextcloud/deck/blob/v1.1.0-beta2/css/deck.scss#L1 - I think they should also be blamed  :sunglasses:

Reply:
Well, sorry if it sounded like blame, that was not the idea.
But you can use this event `OCP\AppFramework\Http\Events\BeforeTemplateRenderedEvent`